app/experimenter/nimbus-ui/src/components/HeaderExperiment/index.stories.tsx

```diff
@@ -0,0 +1,52 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from "react";
+import { storiesOf } from "@storybook/react";
+import { withLinks } from "@storybook/addon-links";
+import HeaderExperiment from ".";
+import { mockExperimentQuery } from "../../lib/mocks";
+import AppLayout from "../AppLayout";
+import { NimbusExperimentStatus } from "../../types/globalTypes";
+
+const { data } = mockExperimentQuery("demo-slug");
+
+storiesOf("components/HeaderExperiment", module)
+  .addDecorator(withLinks)
+  .add("basic", () => (
```

Comment:
Could call this "status: draft", if we ever want to link to this story it'll still be the first/default